Documentation/gettingstarted/bgp.rst

```diff
@@ -14,10 +14,10 @@ BGP provides a way to advertise routes using traditional networking protocols
 to allow Cilium-managed services to be accessible outside the cluster.
 
 This document explains how to configure Cilium's native support for announcing
-``LoadBalancer`` IPs of ``Services`` via BGP. It leverages `MetalLB's
-<https://metallb.universe.tf/>`_ simple and effective implementation of IP
-allocation and the minimal BGP protocol support to do this. The configuration
-for Cilium is the same as MetalLB's configuration.
+``LoadBalancer`` IPs of ``Services`` and a kubernetes node's Pod CIDR range via BGP. 
```

Comment:
@christarazi - updated these docs to mention the pod cidr announcements.

Reply:
Awesome, thanks. 

Minor nit to keep consistency
```suggestion
``LoadBalancer`` IPs of ``Services`` and a Kubernetes node's Pod CIDR range via BGP. 
```